tests/RapidFixFixture/TestFixMessageBuilder.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using RapideFix;
+
+namespace RapideFixFixture
+{
+  internal class TestFixMessageBuilder
+  {
+    private StringBuilder _tags;
```

Comment:
I hate to be the 'incorrect order in file' man. Should we have a convention on that?

Reply:
What do you mean?